pkg/metrics/metrics.go

```diff
@@ -43,6 +43,9 @@ const (
 	backupDeletionSuccessTotal    = "backup_deletion_success_total"
 	backupDeletionFailureTotal    = "backup_deletion_failure_total"
 	backupLastSuccessfulTimestamp = "backup_last_successful_timestamp"
+	backupItemsBackedUpGauge      = "backup_items_backed_up"
+	backupItemsTotalGauge         = "backup_items_total"
```

Comment:
For non-scheduled, manual backups, it's not clear to me what value this metric will provide. Given that any combination of items could be included in manual backups, this gauge value could fluctuate a lot. Do you mostly want to monitor how many items are included in scheduled backups?

Reply:
You're able to add a label to your query to filter the metric. We're using Velero to backup about 700 clusters and we want to monitor the error rate.

Reply:
For example, if you have a `Schedule` you can filter for the label `schedule=schedulename`

Reply:
We discovered some problems using a not correct undeployed cert-manager. These CRs are still deployed, but the webhook isn't accessible anymore. That's why Velero isn't able to backup those CRs and the backup is becoming `PartiallyFailed`